xyz-to-coords.py

```diff
@@ -0,0 +1,38 @@
+import os
+
+def give_coords(filename):
+    directory = r'/home/jacob/scf_guess/geometries'
```

Comment:
This is fine for testing, but it's not going to work on my computer. You need to do this using relative paths so that it is transferable.